golang/pkg/cli/config_file.go

```diff
@@ -192,10 +192,10 @@ func SettingsFileDefaults(initialState *State, args *ArgsFlags) *State {
 		switch {
 		case errors.Is(err, containerRuntime.ErrServerIsOutdated):
 			NotifyOnce("dockerversion", func() {
-				log.Warn().Stack().Err(err).Msg("Server is outdated, please consider updating")
+				log.Warn().Stack().Err(err).Msg("Docker Engine (Server) is outdated, please consider updating")
```

Comment:
This branch and the other can also be reached if one is using Podman.
I would opt for Info log level, since it does not affect application behavior:
```
There is a newer version of the container engine in use, please consider updating.
```